doc/articles/faq.md

```diff
@@ -317,7 +317,7 @@ It's an existing issue, caused by Roslyn. You must add all your TargetFrameworks
 
 That's exactly right, see [this](https://github.com/nventive/Uno/blob/a69879a3154f61b2d493be433aa08bc3b8aa1b06/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L2843) for the XAML-UWP button, and [this](https://github.com/nventive/Uno/blob/a69879a3154f61b2d493be433aa08bc3b8aa1b06/src/Uno.UI/UI/Xaml/Style/Generic/Generic.Native.xaml#L20) for the iOS native button When complete XAML is used for rendering, it's not so much a conversion but more of  a vector rendering. For instance, Uno.UI.iOS uses CGLayer for rendering content it's different because Uno.UI integrates within the layouting system of the platform, which allows for mixed rendering. Uno.UI supports webviews, whereas Flutter does not. Flutter also has to render everything, including what the platform provides by default, such as the Magnifier in the TextBox for accessibility (this is not yet supported either).
 
-## Can I have a screen, use a Map and put markers, using UNO, for Android, and Windows 10?
+## Can I have a screen, use a Map and put markers, using Uno, for Android, and Windows 10?
```

Comment:
I'd rework this line and the next one, the language seems out of place with regards to the rest.

Reply:
The entire FAQ needs to be reworked with a flamethrower, but for now I'm just changing UNO to Uno on FT's request